lib/CustomPropertiesConfig/CustomPropertyFieldEdit.js

```diff
@@ -37,32 +38,35 @@ export default class TermFieldEdit extends React.Component {
       onCancel,
       onSave,
       pickLists,
+      settingsRoute
     } = this.props;
 
+    const translationKey = (settingsRoute === 'terms') ? 'Term' : 'SupplementaryProperty';
```

Comment:
Following on from the other component's comment about using the URL, let's rename this to something more generic. If we end up passing the translation key fragment directly to `CustomPropertiesConfigForm`, we can probably just pass that down here. And maybe clean up the translation keys so we don't sometimes use `terms` and sometimes use `Term` (L55 vs L86)? Just use the opportunity we have now to clean things up.